packages/cubejs-server-core/core/index.d.ts

```diff
@@ -44,6 +44,12 @@ export interface CreateOptions {
 
 export interface OrchestratorOptions {
   redisPrefix?: string;
+  poolOptions?: {
+    poolMin?: number;
+    poolMax?: number;
+    createClient?: (redisUrl: string) => {}
+    destroyClient?: (client) => {}
```

Comment:
I think we should re-use the types from `redis` here:

```suggestion
    createClient?: (redisUrl: string) => RedisClient
    destroyClient?: (client: RedisClient) => void
```